src/api/routes/profileRouter.js

```diff
@@ -0,0 +1,14 @@
+import { Router } from 'express';
```

Comment:
Is there any other router? cause I know you will do even updates on the user profile. I'm asking that because I cant see where you declare a post or a patch